uiServices/src/configServices.go

```diff
@@ -193,3 +195,19 @@ func putConfigs(rw http.ResponseWriter, req *http.Request) {
 
 	rw.WriteHeader(http.StatusNoContent)
 }
+
+func putConfigFileName(rw http.ResponseWriter, req *http.Request) {
+	path := getConfigHeader(req)
+	configFileName := chi.URLParam(req, "configFileName")
+	newConfigFileName := chi.URLParam(req, "newConfigFileName")
+
+	if len(newConfigFileName) < 1 {
+		logrus.Error("File path does not exist")
+		rw.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	os.Rename(fmt.Sprintf("%s%s.xml", path, configFileName), fmt.Sprintf("%s%s.xml", path, newConfigFileName))
```

Comment:
made comments about os.Rename in another PR.

Reply:
@jenny-flynn I can't find your other comment. 


This is the code now 

```
	configFilePath := fmt.Sprintf("%s%s.xml", path, configFileName)
	newConfigFilePath := fmt.Sprintf("%s%s.xml", path, newConfigFileName)

	err := os.Rename(configFilePath, newConfigFilePath)
	if err != nil {
		logrus.Error("File was not updated", err)
		rw.WriteHeader(http.StatusInternalServerError)
		return
	}
```